framework/src/org/checkerframework/framework/util/MultiGraphQualifierHierarchy.java

```diff
@@ -234,12 +235,48 @@ protected void finish(QualifierHierarchy qualHierarchy,
     @SideEffectFree
     @Override
     public String toString() {
-        // TODO: it would be easier to debug if the graph and map were sorted by the key.
```

Comment:
You removed the TODO comment, but this patch does not do the suggested sorting.
Should the TODO comment be retained?